src/router/index.js

```diff
@@ -1,7 +1,7 @@
 import Vue from 'vue'
 import VueRouter from 'vue-router'
 import TheHome from '../views/TheHome.vue'
-import Results from '../views/Results.vue'
+import ResultsContainer from '../views/ResultsContainer.vue'
```

Comment:
Thanks for making this change to the component name.  I think it's easier to distinguish between all of the components related to Results.

Reply:
Agreed! Reading the [Vue Styling Guide](https://vuejs.org/v2/style-guide/) as well, it is a better convention in Vue to have each of the components at least 2 words in PascalCase. This makes it easier for Vue to render any components in kebab-case later as it is built into the functionality of Vue. Thanks for the suggestion for this.

Reply:
Oh, yes! I forgot about that with Vue. Definitely is habit to make things more concise, but using two words in file names is convention with Vue. We probably need to rename `Search.vue` as well.